### PR TITLE
Update the code of disabling installation repos for s390x

### PR DESCRIPTION
**Pull request by tinawang123**
Update the code of disabling installation repos for s390x

- Related ticket: https://progress.opensuse.org/issues/162299
- Verification run: https://openqa.suse.de/tests/14785327
   https://openqa.suse.de/tests/14785383#

### CODE REVIEW

**Comment by hjluo** [comment]
LGTM

File: lib/migration.pm [hunk 1]
```diff
@@ -151,7 +151,9 @@ sub remove_dropped_modules_packages {
 sub disable_installation_repos {
     if (is_s390x) {
         my $repos = script_output("zypper lr -u | awk '/ftp:.*?openqa.suse.de|10.145.10.207|10.160.0.100/ {print \$1}'");
```
Comment (lemon-suse) [inline lib/migration.pm:153]: It seems the output with BR which caused the failure, could we update regexp to replace BR with space then don't need add code to deal with each line?

Comment (tinawang123) [reply lib/migration.pm:153]: As the output , we need remove those three repos. https://openqa.suse.de/tests/14785327#step/patch_sle/20
Is it expected? @hjluo

Comment (tinawang123) [reply lib/migration.pm:153]: Is remove BR, it will change to '123'. 
 zypper -n mr -d 123
Repository 123 not found.

Comment (lemon-suse) [reply lib/migration.pm:153]: Shouldn't remove but replace BR with space, that output should be '1 2 3'.

Comment (tinawang123) [reply lib/migration.pm:153]: https://openqa.suse.de/tests/14800054#step/patch_sle/16

File: lib/migration.pm [hunk 2]
```diff
@@ -151,7 +151,8 @@ sub remove_dropped_modules_packages {
...
```
Comment (lemon-suse) [inline lib/migration.pm:153]: Just add the "| tr '\n' ' '" at the end this cmd line then got the output separate with space, you don't need change other code then.

Comment (tinawang123) [reply lib/migration.pm]: VR: https://openqa.suse.de/tests/14809451#

File: lib/migration.pm [hunk 3]
```diff
@@ -150,8 +150,8 @@ sub remove_dropped_modules_packages {
 # https://documentation.suse.com/sles/15-SP2/html/SLES-all/cha-upgrade-online.html#sec-upgrade-online-zypper
 sub disable_installation_repos {
     if (is_s390x) {
-        my $repos = script_output("zypper lr -u | awk '/ftp:.*?openqa.suse.de|10.145.10.207|10.160.0.100/ {print \$1}'");
-        zypper_call "mr -d $repos" if (length $repos);
+        my $repos = script_output("zypper lr -u | awk '/ftp:.*?openqa.suse.de|10.145.10.207|10.160.0.100/ {print \$1}' | tr '\n' ' '");
+        zypper_call "mr -d $repos";
```
Comment (lemon-suse) [inline lib/migration.pm:154]: Please reserve the condition of "if (length $repos) " which used to ensure the repos is not a NULL.

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**